bin/ci-test

```diff
@@ -37,7 +37,9 @@ function build() {
   export CORE=dependabot/dependabot-core
   export UPDATER=dependabot/dependabot-updater
 
+  # shellcheck disable=SC2034  # shellcheck isn't smart enough to realize `docker build` uses `TARGETARCH`
   local TARGETARCH;
+  # shellcheck disable=SC2034  # shellcheck isn't smart enough to realize `docker build` uses `TARGETARCH`
```

Comment:
I had to repeat this line because I couldn't find a way to disable/re-enable shellcheck for a block of code... it was only the line or the file AFAICT.

Reply:
Technically `TARGETARCH` should be local to this function, but I don't think using a standard global variable (by removing the `local` line) should cause any issues, and would save us from repeating this line which is a bit confusing.

Alternatively, we could add a blank line above this line just to emphasize that each directive applies to the line below it and this one is not just a mistyped `# shellcheck enable=SC2034` (which I just checked it's not valid, but it could!).

Reply:
Yeah, I'm not too worried once it's in the code about being mistaking for an accident... if someone tries to "clean it up", then CI will squawk at them once #5820 lands.

Reply:
> Yeah, I'm not too worried once it's in the code about being mistaking for an accident... if someone tries to "clean it up", then CI will squawk at them once https://github.com/dependabot/dependabot-core/pull/5820 lands.

Even if our CI would catch it, if we can spare people the time to create that wrong change, we should try!

Reply:
I know I left this comment after approving, because I noticed the duplicated line late, but if we can add that blank line or remove the `local TARGETARCH` line I'd go with that. But no blocker at all.

Reply:
Sorry, just seeing this. I misread your comment earlier and didn't realize I could potentially remove the `local TARGETARCH` line. Let me tag this into #5820